docs/en/connector-v2/changelog/connector-hugegraph.md

```
@@ -0,0 +1,7 @@
+<details><summary> Change Log </summary>
+
+| Change | Commit | Version |
+| --- | --- |---------|
+|[Feature][Connector-V2] Support sink connector for Apache HugeGraph|https://github.com/apache/seatunnel/pull/10002/commits/002a653d11f48c3f76b47db23f5f2a68bc9d690c| 2.3.12  |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)